pinot-core/src/main/java/org/apache/pinot/core/operator/blocks/IntermediateResultsBlock.java

```diff
@@ -117,6 +119,19 @@ public IntermediateResultsBlock(AggregationFunction[] aggregationFunctions,
     _dataSchema = dataSchema;
   }
 
+  /**
+   * Constructor for aggregation group-by order-by result with {@link AggregationGroupByResult}.
+   */
+  public IntermediateResultsBlock(AggregationFunction[] aggregationFunctions,
+                                  @Nullable AggregationGroupByResult aggregationGroupByResults,
```

Comment:
Follow https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup to setup the code style in your IDE

Reply:
Followed